ci/kind/config-3nodes.yml

```diff
@@ -5,5 +5,11 @@ networking:
   podSubnet: 10.10.0.0/16
```

Comment:
This file is not used by `hack/make-metrics-doc.sh`, so how does it work?

Reply:
@antoninbas i guess by mistake I have pushed my dummy changes the actual were made to .ci/kind)kind-setup.sh file